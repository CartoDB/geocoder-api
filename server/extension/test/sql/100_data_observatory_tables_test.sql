SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_connectusertable'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, text, text, text');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_getreturnmetadata'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_fetchjoinfdwtabledata'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, text, text, json');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_disconnectusertable'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, text, text');

