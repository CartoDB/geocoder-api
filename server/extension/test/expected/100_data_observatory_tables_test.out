SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_connectusertable'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, text, text, text');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_getreturnmetadata'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_fetchjoinfdwtabledata'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, text, text, json');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = '_dst_disconnectusertable'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, text, text');
 exists 
--------
 t
(1 row)

